static/usage/searchbar/debounce/react.md

```diff
@@ -0,0 +1,42 @@
+```tsx
+import React from 'react';
+import { IonItem, IonList, IonSearchbar } from '@ionic/react';
+
+function Example() {
+  const handleChange = (ev: Event) => {
+    let query = "";
+    const target = ev.target as HTMLIonSearchbarElement;
+    if (target) query = target.value!.toLowerCase();
+
+    const items = Array.from(document.querySelector('ion-list')!.children) as HTMLElement[];
+
+    requestAnimationFrame(() => {
```

Comment:
Why the `requestAnimationFrame`? (Ditto for the other frameworks)

Reply:
I just copied it from some old usage, looks like it isn't needed anymore, will remove.